### PR TITLE
tweak NO_RFC7540 text

### LINKED ISSUE #1765: Clarify how endpoints use NO_RFC7540 setting
Paraphrasing feedback from @rmarx, the section explains the setting just fine but could be clearer on how the client or server might use this setting

### PR DESCRIPTION
**Pull request by LPardue**
Fixes #1765.

Minimal change to put focus on what endpoint do with this.

### CODE REVIEW

**Comment by LPardue** [comment]
That doesn't quite do it for me. I'll do so more thinking on how we can editorialise this part.

**Comment by LPardue** [comment]
I've tried to incorporate the feedback and @kazuho's suggestion. This doesn't introduce any new requirements, so remains editorial.

File: draft-ietf-httpbis-priority.md [hunk 1]
```diff
@@ -210,24 +210,30 @@ The problems and insights set out above provided the motivation for deprecating
...
+SETTINGS frame. Senders MUST NOT change the SETTINGS_NO_RFC7540_PRIORITIES value
 after the first SETTINGS frame. Receivers that detect a change MAY treat it as a
 connection error of type PROTOCOL_ERROR.
 
-The SETTINGS frame precedes any HTTP/2 priority signal sent from a client, so a
-server can determine if it needs to allocate any resource to signal handling
-before they arrive. A server that receives SETTINGS_NO_RFC7540_PRIORITIES
-with value of 1 MUST ignore HTTP/2 priority signals.
+Clients can send SETTINGS_NO_RFC7540_PRIORITIES with a value of 1 to indicate
+they are not using HTTP/2 priority signals. The SETTINGS frame precedes any
```
Comment (kazuho) [inline draft-ietf-httpbis-priority.md:225]:
```suggestion
that they are not using HTTP/2 priority signals. The SETTINGS frame precedes any
```

**Review by kazuho — APPROVED** [review]
Thank you @LPardue. LGTM modulo purely-editorial, take-it-or-leave-it suggestion.

**Comment by rmarx** [comment]
I think this is a very clean solution @LPardue!

It's not 100% in its description (e.g., there's no "client SHOULD NOT send old-style signals if setting is 1", as you say above that's difficult to cover fully), but the missing stuff can be deduced from the rest (i.e., even if the client does send those, server will ignore).

**Comment by LPardue** [comment]
> It's not 100% in its description (e.g., there's no "client SHOULD NOT send old-style signals if setting is 1", as you say above that's difficult to cover fully), but the missing stuff can be deduced from the rest (i.e., even if the client does send those, server will ignore).

yeah that's covered in section 2.1.1 that deals specifically with what a client should do when using extensible priorities. At this general layer of disabling, I'm happy not to state any requirement.

**Pull request merged**